### PR TITLE
Bump build-tools and switch to 00-tqu

### PR DESCRIPTION
**Pull request by stdweird**
Requires new release of build-tools (incl https://github.com/quattor/maven-tools/pull/119)

### CODE REVIEW

**Comment by stdweird** [comment]
It is still missing the version bump in pom.xml; bump this is good example of what is to come.

**Comment by stdweird** [comment]
@ned21 adapted to severity 5

File: src/main/perl/ComponentProxyList.pm [hunk 1]
```diff
@@ -58,55 +58,55 @@ sub reportComponents
...
-# Run the pre-config $hook, possibly timing out after $timeout seconds
+
+# common code for pre/post hooks
+# Run the pre/post-config $hook, possibly timing out after $timeout seconds
+# If data is defined, it is passed via stdin in JSON format
 # Returns 1 on success, 0 on failure
-sub pre_config_actions
+# TODO: the types are probably not preserved all the way here,
+#   so the JSON format e.g. might have strings for integers
+sub _config_actions
```
Comment (ned21) [inline src/main/perl/ComponentProxyList.pm]: How does this refactoring relate to the Perl warnings during tests? Should this be a separate commit?

Comment (stdweird) [inline src/main/perl/ComponentProxyList.pm]: hmmm, i wasn't going to fix the same issue twice, and the code was copy/paste. so maybe the refactor should be done in separate commit. let me know if you insist...

Comment (ned21) [inline src/main/perl/ComponentProxyList.pm]: Oh - I see why this was necessary now. I know last time you had to split a commit it was super-painful but in this case it's just one file so should be relatively easy? I definitely think it would be better as two commits.

**Comment by stdweird** [comment]
@ned21 the painfullness is only due to number of commits stack on top of each other. this was okish 😄

**Comment by ned21** [comment]
Thanks! LGTM. I guess we are just waiting on a new release of build tools.

**Comment by stdweird** [comment]
@ned21 @jrha alos read to go
